2021/03.md

```diff
@@ -82,7 +82,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
     |   | 3     | 30m     | [Intl.Segmenter](https://github.com/tc39/proposal-intl-segmenter) for Stage 4 ([slides](https://docs.google.com/presentation/d/1tkyQVE3o5qpbbJ39RidyZiy-r179RXraOKDeWLB5RB8/edit?usp=sharing), [spec text](https://tc39.es/proposal-intl-segmenter/)) | Richard Gibson |
     |   | 3     | 30m     | [top-level await update towards stage 4](https://github.com/tc39/proposal-top-level-await), DFS ordering invariant normative change ([PR](https://github.com/tc39/proposal-top-level-await/pull/159)) and bug fixes ([PR](https://github.com/tc39/proposal-top-level-await/pull/161)) | Yulia Startsev |
     |   | 3     | 15m     | [Class `static` Initialization Blocks](https://github.com/tc39/proposal-class-static-block) Stage 3 (conditional) Update ([slides](https://1drv.ms/p/s!AjgWTO11Fk-TkfhG_gVnKlNwMT-MyA?e=owLLRf)) | Ron Buckton |
-    |   | 2     | 120m    | [Temporal](https://github.com/tc39/proposal-temporal) for Stage 3 | Philip Chimento |
+    |   | 2     | 120m    | [Temporal](https://github.com/tc39/proposal-temporal) for Stage 3 ([slides](http://ptomato.name/temporal-slides-in-progress/)) | Philip Chimento |
```

Comment:
is there an https url we could use?

Reply:
OK, the rest of my site is broken now, but this URL should work

Reply:
oh noes, i didn't mean to cause trouble! thanks, i'll land this